src/sinks/influxdb/logs.rs

```diff
@@ -225,14 +225,12 @@ impl InfluxDBLogsConfig {
     }
 }
 
-impl Value {
-    fn to_field(&self) -> Field {
-        match self {
-            Value::Integer(num) => Field::Int(*num),
-            Value::Float(num) => Field::Float(*num),
-            Value::Boolean(b) => Field::Bool(*b),
-            _ => Field::String(self.to_string_lossy()),
-        }
+fn value_to_field(value: &Value) -> Field {
+    match value {
+        Value::Integer(num) => Field::Int(*num),
+        Value::Float(num) => Field::Float(*num),
+        Value::Boolean(b) => Field::Bool(*b),
+        _ => Field::String(value.to_string_lossy()),
     }
 }
```

Comment:
Could this be `impl From<&Value> for Field`? `enum Field` is defined within Vector.

Reply:
I've reverted these changes, since `Value` was moved back into Vector again 👍